src/assets/icons/food/icon-taco.vue

```diff
@@ -150,4 +129,4 @@ import { defineComponent } from "vue";
 export default defineComponent({
   name: "TacoIcon"
```

Comment:
i think it's `IconTaco`

Reply:
Right, I also changed that for the other icons